contributor-guide/modules/ROOT/pages/contributors-faq.adoc

```diff
@@ -0,0 +1,129 @@
+////
+Copyright (c) 2024 The C++ Alliance, Inc. (https://cppalliance.org)
+
+Distributed under the Boost Software License, Version 1.0. (See accompanying
+file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+Official repository: https://github.com/boostorg/website-v2-docs
+////
+= Contributors Frequently Asked Questions
+:navtitle: Contributors FAQ
+
+This section contains answers to the common questions that new contributors to Boost often have.
+
+== Artificial Intelligence
+
+. *If I wanted to build a library that focused on the lower-level building blocks required by generative AI, what functional areas should I consider?*
++
+Consider one or more of the following:
+
+* Provide implementations of various _neural network architectures_ commonly used in generative AI, such as convolutional neural networks (CNNs), recurrent neural networks (RNNs), generative adversarial networks (GANs), variational autoencoders (VAEs), and transformers. These architectures serve as the backbone for generating complex and realistic outputs.
++
+* Develop _optimization algorithms_ for training neural networks, such as stochastic gradient descent (SGD), Adam, RMSprop, and other advanced optimization techniques. Efficient optimization algorithms are crucial for training AI models effectively and achieving good performance.
++
+* Offer a variety of _loss functions_ suitable for different generative tasks, including image generation, text generation, and sequence generation. Loss functions measure the discrepancy between the model's predictions and the ground truth and play a crucial role in guiding the training process.
++
+* Provide utilities for _data augmentation_ techniques that can enhance the diversity and robustness of training data. Data augmentation is particularly important for generative tasks where having a large and diverse dataset is essential for training high-quality models.
++
+* Implementations of various _probability distributions_ are used in generative modeling, such as Gaussian distributions, categorical distributions, and multinomial distributions. Probability distributions are fundamental for modeling uncertainty and generating diverse outputs.
++
+* Offer _evaluation metrics_ for assessing the performance of generative AI models, such as inception score, Frechet Inception Distance (FID), perplexity, and BLEU score. These metrics provide quantitative measures of the quality, diversity, and realism of generated outputs.
++
+* Support _parallelization techniques_ and _GPU acceleration_ to speed up the training and inference process, especially for large-scale generative models that require significant computational resources.
++
+* Provide tools for deploying trained generative _AI models_ into production environments, including integration with deployment platforms, model serialization formats, and optimization techniques for inference speed and resource efficiency.
+
+== Development Environments
```

Comment:
I think some people just use visual studio. (not visual studio code)

Reply:
True, added a reference to VS